recipe/meta.yaml

```diff
@@ -42,7 +41,9 @@ test:
   commands:
     - if not exist %LIBRARY_INC%\\symengine\\basic.h exit 1     # [win]
     - if not exist %LIBRARY_LIB%\\symengine.lib exit 1          # [win]
+    {% if version != "0.12.0" %}
     - if not exist %LIBRARY_BIN%\\symengine-{{ maj_min }}.dll exit 1          # [win]
+    {% endif %}
```

Comment:
This was because the version was not updated in CMakeLists.txt before releasing 0.12.0. I fixed it now, but we should release a 0.13.0 for both C++ and python libraries.

Reply:
I saw that. Sorry about that. Thanks!